interface.go

```diff
@@ -38,7 +36,34 @@ type ElasticProcessor interface {
 
 // FeesProcessorHandler defines the interface for the transaction fees processor
 type FeesProcessorHandler interface {
-	ComputeGasUsedAndFeeBasedOnRefundValue(tx process.TransactionWithFeeHandler, refundValueStr string) (uint64, *big.Int)
-	ComputeTxFeeBasedOnGasUsed(tx process.TransactionWithFeeHandler, gasUsed uint64) *big.Int
-	ComputeMoveBalanceGasUsed(tx process.TransactionWithFeeHandler) uint64
+	ComputeGasUsedAndFeeBasedOnRefundValue(tx coreData.TransactionWithFeeHandler, refundValue *big.Int) (uint64, *big.Int)
+	ComputeTxFeeBasedOnGasUsed(tx coreData.TransactionWithFeeHandler, gasUsed uint64) *big.Int
+	ComputeGasLimit(tx coreData.TransactionWithFeeHandler) uint64
+	IsInterfaceNil() bool
+}
+
+// Coordinator defines what a shard state coordinator should hold
+type Coordinator interface {
```

Comment:
please rename to `ShardCoordinator` as we also have `NodesCoordinator`
Before, it was accessed by `sharding.Coordinator` which made sense. But now, it doesn't

Reply:
done.